x-pack/plugins/observability_solution/infra/server/saved_objects/custom_dashboards/custom_dashboards_saved_object.ts

```diff
@@ -42,7 +62,13 @@ export const infraCustomDashboardsSavedObjectType: SavedObjectsType = {
     '1': {
       changes: [],
       schemas: {
-        create: schema.object(createSchema),
+        create: schema.object(createV1Schema),
+      },
+    },
+    '2': {
+      changes: [],
+      schemas: {
+        create: schema.object(createV2Schema),
       },
```

Comment:
I think there has been a misunderstanding. You added mappings in that PR, so the associated model version must reflect those additions, via a `mappings_addition` type of change.

```ts
 '2': {
      changes: [
         { type: 'mappings_addition', addedMappings: theMappingsThatWereAdded }
      ],
      schemas: {
        create: schema.object(createV2Schema),
      },
}
```